### PR TITLE
Stylecheck and Coverage to Tox

### LINKED ISSUE #38: Add PEP 8 and coveragepy support to tox.ini

### PR DESCRIPTION
**Pull request by isidentical**
Stylecheck and Coverage to Tox
Fixes #38

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+envlist = 
+    py{27, 34, 35, 36, 37, 38, py, py3.5}
+    flake8
 skipsdist = True
+skip_missing_interpreters = true
 
 [testenv]
 usedevelop = True
-commands = nosetests -v --nocapture {posargs}
+commands = nosetests --with-coverage -v --nocapture {posargs}
```
Comment (berkerpeksag) [inline tox.ini]: The recommended way to use coverage.py and nose is documented at https://coverage.readthedocs.io/en/latest/faq.html

We also need to create a `.coveragerc` file to 1) exclude some files (for example `astor/codegen.py`) from the report 2) add missing branch information to the report.

File: tox.ini [hunk 2]
```diff
@@ -1,10 +1,20 @@
 [tox]
-envlist = py27, py34, py35, py36, py37, py38, pypy, pypy3.5
+envlist = 
+    py{27, 34, 35, 36, 37, 38, py, py3.5}
+    flake8
```
Comment (berkerpeksag) [inline tox.ini]: flake8 environment doesn't show up on Travıs CI. You probably need to update `.travis.yml` as well.

I'd also rename it to 'lint'.

**Comment by isidentical** [comment]
Now travis lints under python3.7 environment.

**Comment by isidentical** [comment]
> Can you add
Done
> Also, why did you change permission of `tests/build_expressions.py`?
Sorry for inconvenience, i think i made that accidentally.

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**